debezium-testing/debezium-testing-system/src/main/java/io/debezium/testing/system/tools/kafka/builders/FabricKafkaConnectBuilder.java

```diff
@@ -112,7 +112,7 @@ public FabricKafkaConnectBuilder withBuild(List<Plugin> plugins) {
                 .editSpec()
                 .withNewBuild()
                 .withNewImageStreamOutput()
-                .withImage("testing-openshift-connect:latest")
```

Comment:
I'd keep it as it is - as we discussed during the meeting, we actually create the image here and it's push to OCP registry. Any attempt pull/push to dockerhub would fail anyway as it's missing the organization.